### DISCUSSION TITLE
Using Tasker to connect VPN

### DISCUSSION THREAD

**Discussion by pixbalance** [schwabe/ics-openvpn#1532] [Q&A] [answered]
I am searching for a while how to connect the app via tasker. So far I am using the App OpenVPN Tasker Plugin, but I I was hoping to find someone who hast solved it with Send Intent or in another way.

In tasker I have created a task with

action: android.intent.action.MAIN
extra: de.blinkt.openvpn.api.profileName:{PROFILNAME}
package: de.blinkt.openvpn
class: de.blinkt.openvpn.ConnectVPN
target: activity

but unfortunately nothing is happening.

**Comment by remoter3406** [comment, marked as answer]
If someone have the same problem:
class: de.blinkt.openvpn.**api**.ConnectVPN is missing
